api/src/main.py

```diff
@@ -260,7 +261,9 @@ async def publish_dataset(dataset: schemas.Dataset):
         },
         "links": [],
         "type": "Collection",
-        "dashboard:time_density": dataset.time_density,
+        "dashboard:time_density": dataset.time_density
+        if dataset.time_density
+        else "null",
```

Comment:
I think you want to fix this further upstream--around line 61, the default should be None which will be converted to json null